plugin/storage/es/factory.go

```diff
@@ -171,6 +173,8 @@ func (f *Factory) Initialize(metricsFactory metrics.Factory, logger *zap.Logger)
 		}
 	}
 
+	f.templateBuilder = es.TextTemplateBuilder{}
```

Comment:
If this is a stateless class that requires no parameters to initialize, why pass it around? It can be instantiated where needed.

Reply:
I reverted the code back to its original state. https://github.com/jaegertracing/jaeger/commit/eaa6bdce979d6f4779ffe52df358a43fcf5ac38a